src/connectors/stationhead.ts

```diff
@@ -0,0 +1,4 @@
+export {};
+
+Connector.useMediaSessionApi();
+Connector.playerSelector = '#root>div>div:nth-child(2)>div>div>div>div>div:nth-child(2)>div:nth-child(2)>div';
```

Comment:
Is there not any class or ID on the element to make this less brittle to changes? 

Also please add a new line after :)

Reply:
The website only have generated class from library they use, when website have an update will regenerate the class. 
![image](https://github.com/web-scrobbler/web-scrobbler/assets/18456011/02857253-c6a3-48c9-afe4-853ca498ce69)

Done add new line on end of file.